src/harness/compilerRunner.ts

```diff
@@ -50,7 +50,8 @@ class CompilerBaselineRunner extends RunnerBase {
     }
 
     private makeUnitName(name: string, root: string) {
-        return ts.isRootedDiskPath(name) ? name : ts.combinePaths(root, name);
+        const path = ts.toPath(name, root, (fileName) => Harness.Compiler.getCanonicalFileName(fileName));
+        return path.replace(Harness.IO.getCurrentDirectory(), "/");

```

Comment:
`getCurrentDirectory` should also be converted to `path` otherwise `replace` might work incorrectly in case of different casing